### ISSUE TITLE
Entoptic phenomenon

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10588]
**Preferred term label:**

**Synonyms**
Entoptic phenomenona

**Definition (free text, please give PubMed ID)**
https://www.oxfordreference.com: Visual sensations caused by changes within the eye itself, rather than by the normal light stimulation process. The commonest are tiny floating spots (floaters) that most people can see occasionally, especially when gazing at a brightly illuminated background (such as a blue sky).

PMID: 36788360 Entoptic phenomena occur as observations of physiologic processes from the optic apparatus itself and that are present in great magnitude in VSS patients. They are classified in the syndrome as floaters, blue field entoptic phenomenon, photopsia and self-light of the eye

**Parent term (use hpo.jax.org/app)**
Abnormality of vision HP:0000504

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:420556 Visual snow syndrome

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 

Adding new term: 
- ID: HP:6001103
- label: Entoptic phenomenon
- parents: [Abnormality of vision (HP:0000504)](https://hpo.jax.org/app/browse/term/HP:0000504)
- pmid: PMID:36788360
- def: Visual sensations caused by changes within the eye itself, rather than by the normal light stimulation process. Entoptic phenomena are classified in the syndrome as floaters, blue field entoptic phenomenon, photopsia and self-light of the eye.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10588
- synonyms: Entoptic phenomenona(NONE)
